mockwebserver/src/main/kotlin/mockwebserver/MockWebServer.kt

```diff
@@ -0,0 +1,1149 @@
+/*
+ * Copyright (C) 2011 Google Inc.
+ * Copyright (C) 2013 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package mockwebserver
```

Comment:
I think we want mockwebserver2 or mockwebserver3. It’s definitely uglier! But I hate trying to remember whether org.junit or junit.framework is the right one, and this is similarly ambiguous.

Reply:
I actually like it.  mockwebserver3 matches okhttp3.